File: Day 5/javascript/calculator/compound_interest.js
// Compound Interest:
// Input: time, rate and principal
// Output: interest

function compoundInterest(p, r, t, n){
    var amount = p * (Math.pow((1 + (r / n)), (n * t)));
    var interest =amount-p;
    return (interest);
}
console.log(compoundInterest(2000,0.08,5,12));

File: Day 5/python/calculator/compound_interest.py
# Compound Interest:
# Input: time, rate and principal
# Output: interest

def compound_interest(p, r, t, n):
    amount = p*(pow((1+r/n),(n*t)))
    ci = amount-p
    return ci
p=int(input('The principal is: '))
t=float(input('The time period is: '))
r=float(input('The rate of interest is: '))
n=int(input('Number of times: '))
print(compound_interest(p, r, t, n))